### DISCUSSION TITLE
How to install an extension for Green Browser extension lesson

### DISCUSSION THREAD

**Discussion by tashasyarifah** [microsoft/Web-Dev-For-Beginners#691] [Ideas]
Hi everyone, I have a suggestion for the extension lesson. This might be helpful for someone who has never used an extension in any browser like me. 

In the lessons, you mentioned the steps clearly but I couldn't show up my extension in my browser. I was wondering what could go wrong. Then I realized something. 

![A picture of the Microsoft edge extension page that has a square green on the left bottom. There are two toggles inside the square green; developer mode and allow extension from other stores.](https://user-images.githubusercontent.com/66051012/173991480-984833c3-50c4-42c1-a357-f22a16baf85a.png)

The picture above shows that my developer mode and allow extension from other stores toggles are off.  This is the reason why my extension didn't show. Those two toggles are off by default, so it needs to be on manually. When I look again at the lesson, I realized that it was already on for both toggles.

I would to suggest that you mentioned those toggles in the lessons. It will be helpful. 

Thank you!

**Comment by saragibby** [comment]
Thank you @tashasyarifah for pointing this out! I will be sure to get the lesson update to include this.
